db/migrate/20151110191852_create_redirections.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRedirections < ActiveRecord::Migration
+  def change
+    create_table :redirections do |t|
+      t.timestamps null: false
+      t.references :next
+      t.string :slug, index: true
+      t.string :url, length: 255
```

Comment:
Maybe i'm mis-remembering, but I thought the default length of a string was 128 char? Can't find it now, but if so, this will make sure it's longer than that because we can has some whack-ass long urls.